pkg/virt-operator/kubevirt.go

```diff
@@ -1040,6 +1040,7 @@ func (c *KubeVirtController) syncInstallation(kv *v1.KubeVirt) error {
 			logger.Info("All KubeVirt components ready")
 			kv.Status.Phase = v1.KubeVirtPhaseDeployed
 			util.UpdateConditionsAvailable(kv)
+			kv.Status.ObservedGeneration = &kv.ObjectMeta.Generation
```

Comment:
if this is primarily for testing purposes, can we consider an annotation instead for now?

Reply:
I don't see it primarily for testing. We have right now no clear way of  telling if changes got applied. `observedGeneration` is a very simple and common mechanism to do it.

Reply:
I see it more like the tests showing us a shortcoming on our API. Thoughts?

Reply:
so, you'd expect something like the hco to consume this?